pimcore/models/Element/AbstractElement.php

```diff
@@ -138,7 +138,11 @@ public function isLocked()
     public function getUserPermissions()
     {
         $elementType = Service::getElementType($this);
-        $vars = get_class_vars("\\Pimcore\\Model\\User\\Workspace\\" . ucfirst($elementType));
+        if ($elementType === 'object'){
```

Comment:
We should backport and use `\Pimcore\Model\Element\Service::getBaseClassNameForElement()` from v5 here. 
https://github.com/pimcore/pimcore/blob/856b7649911c8f8e571901e883ec47e8c035163c/models/Element/AbstractElement.php#L174
https://github.com/pimcore/pimcore/blob/8808f7342acca59e56aa34bf9dac47979d11fce1/models/Element/Service.php#L269